extensions/standard-processors/tests/unit/GetFileTests.cpp

```diff
@@ -52,7 +52,7 @@ TEST_CASE("GetFile: MaxSize", "[getFileFifo]") {  // NOLINT
   std::string in_file = temp_path + utils::file::FileUtils::get_separator() + "testfifo";
 
   // Define test input file
-  std::string hidden_in_file(in_dir);
+  std::string hidden_in_file(temp_path);
```

Comment:
Are you sure it's wrong? It seems right to me.

If you look closely `in_dir` is just a template that is used by `createTempDirectory()` to generate a unique dir name, which is then assigned to `temp_path`.

The `in_file` path was already built using `temp_path`, but `hidden_in_file` was using `in_dir` instead, which seems wrong to me and would cause both files to be at different locations.

The test never failed because the Processor would not get the file either way since it ignores hidden files.

There's one more improvement to be done here: both file paths are constructed in different ways, which just makes it confusing. I can make them consistent, following your suggestion above, but using `temp_path` instead:

```
// Define test input files
std::string in_file(temp_path + utils::file::FileUtils::get_separator() + "testfifo");
std::string hidden_in_file(temp_path + utils::file::FileUtils::get_separator() + ".testfifo")
```

@arpadboda Let me know your thoughts.

Reply:
I definitely like your suggestion, please proceed with that!

Reply:
Done!